src/SFA.DAS.EmployerUsers.Api/Controllers/SearchController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Threading.Tasks;
+using System.Web.Http;
+using SFA.DAS.EmployerUsers.Api.Orchestrators;
+
+namespace SFA.DAS.EmployerUsers.Api.Controllers
+{
+    [RoutePrefix("api/users/search")]
+    public class SearchController : ApiController
+    {
+        private readonly UserOrchestrator _orchestrator;
+
+        public SearchController(UserOrchestrator orchestrator)
```

Comment:
I am not sure orchestrators are adding any value here, however if we are going to us them then we should follow the 1:1 controller:orchestrator pattern - https://www.simple-talk.com/dotnet/asp-net/never-mind-the-controller-here-is-the-orchestrator/